pallets/author-mapping/src/lib.rs

```diff
@@ -32,29 +32,167 @@ pub use pallet::*;
 pub mod pallet {
 
 	use frame_support::pallet_prelude::*;
+	use frame_support::traits::ReservableCurrency;
 	use frame_system::pallet_prelude::*;
 	use nimbus_primitives::AccountLookup;
+	use sp_runtime::Percent;
+
+	/// The portion of the security deposit that goes to the the account who reports it occupying
+	/// space after it should have been cleaned or rotated.
+	pub const NARC_REWARD: Percent = Percent::from_percent(5);
```

Comment:
what does "NARC" stand for ? 😆

Reply:
Lol. In middle school I heard that word a lot. Kids used it to mean "tattle tale" or "tell the teacher". For example: "I got detention because Billy narced on me for running in the hall".

Today I learned its etymology 😅

Reply:
Oh yeah I know what it means ^^ I was just wondering why you were using it here

Reply:
I was thinking of one account "narcing" on another account for not cleaning up its storage. "Teacher, billy registered an authorid and didn't clean up when he was done."